resources/secure/apps/spire-debug.yaml

```diff
@@ -1,16 +1,3 @@
-# apiVersion: v1
-# kind: ConfigMap
-# metadata:
-#   name: path-to-inputfile
-# data:
-#   inputfile.txt: |
-#     db-config/config.ini
----
-apiVersion: v1
```

Comment:
Oh, I see, so you are assuming the SA is already crated since we did it for the App earlier. OK that makes sense

Reply:
the issue is when you delete the resources associated with the debug, it also deletes the SA